nixos/modules/programs/environment.nix

```diff
@@ -20,6 +20,7 @@ in
       { NIXPKGS_CONFIG = "/etc/nix/nixpkgs-config.nix";
         PAGER = mkDefault "less -R";
         EDITOR = mkDefault "nano";
+        XCURSOR_PATH = "$HOME/.icons";
```

Comment:
This seems a little out of place at the very top level. Many of us don't even use a graphical system. All my NixOS boxes are headless EC2 instances. This doesn't really hurt anything, but it probably belongs in a different module?